Modules/Core/Common/include/itkMacro.h

```diff
@@ -1416,31 +1381,80 @@ itkDynamicCastInDebugMode(TSource x)
 #endif
 }
 
-#ifdef ITK_LEGACY_REMOVE
-#  if __cplusplus >= 202002L
-#    define ITK_NODISCARD(message) [[nodiscard(message)]]
-#  else
-#    define ITK_NODISCARD(message) [[nodiscard]]
-#  endif
+#if __cplusplus >= 202002L
+#  define ITK_NODISCARD(message) [[nodiscard(message)]]
 #else
-#  define ITK_NODISCARD(message)
+#  define ITK_NODISCARD(message) [[nodiscard]]
 #endif
 
+//-*-*-*
```

Comment:
Does this comment have some special meaning?

Reply:
It was there before moved with code from above.